admin-ui/app/redux/features/types/authTypes.ts

```diff
@@ -41,4 +41,5 @@ export interface AuthState {
   loadingConfig: boolean
   authState?: any
   userInum?: string | null
+  hasFetchUserInformation: boolean
```

Comment:
Name the flag to reflect past-tense state (readability).

The current name is ambiguous. Prefer isUserInfoFetched (or hasFetchedUserInfo) to clearly mean “already fetched”.

Apply:

```diff
-  hasFetchUserInformation: boolean
+  isUserInfoFetched: boolean
```

Follow through in the reducer/selectors/usages in this PR.